demos/python_demos/gesture_recognition_demo/README.md

```diff
@@ -67,26 +72,26 @@ To run the demo, you can use public or pre-trained models. To download the pre-t
 
 > **NOTE**: Before running the demo with a trained model, make sure the model is converted to the Inference Engine format (`*.xml` + `*.bin`) using the [Model Optimizer tool](https://docs.openvinotoolkit.org/latest/_docs_MO_DG_Deep_Learning_Model_Optimizer_DevGuide.html).
 
-To run the demo, please provide paths to the ASL recognition and person detection models in the IR format, to a file with class names, and to an input video:
+To run the demo, please provide paths to the gesture recognition and person detection models in the IR format, to a file with class names, and to an input video:
 ```bash
-python asl_recognition_demo.py \
+python gesture_recognition_demo.py \
 -m_a /home/user/asl-recognition-0003.xml \
```

Comment:
0003 model was replaced with 0004. Could you update it while you are here?

Reply:
Renamed